lib/happo/public/happo-styles.css

```diff
@@ -1,4 +1,11 @@
 body {
   background-color: #f0f0f0;
   font-family: helvetica, arial;
+  background: repeating-linear-gradient(
+    -45deg,
+    #f0f0f0,
+    #f0f0f0 10px,
+    #f4f4f4 10px,
+    #f4f4f4 20px
+  );
```

Comment:
What do you think about a checkerboard pattern instead? That's traditionally what is used for this sort of thing.

```css
  background-color: #ebebeb;
  background-image: linear-gradient(45deg, #f4f4f4 25%, transparent 25%, transparent 75%, #f4f4f4 75%, #f4f4f4), 
  linear-gradient(45deg, #f4f4f4 25%, transparent 25%, transparent 75%, #f4f4f4 75%, #f4f4f4);
  background-size:20px 20px;
  background-position:0 0, 10px 10px
```